src/main/minio.js

```diff
@@ -403,7 +403,6 @@ export class Client {
           this.logHTTP(reqOptions, null, e)
           cb(e)
         })
-      req.once('finish', () => pipe.removeAllListeners('error'));
```

Comment:
Yes this was the patch provided by @dcharbonnier . I do remember the context. Let me find. @dcharbonnier do you remember?